tests/Unit/ApparentHorizons/Test_Strahlkorper.cpp

```diff
@@ -0,0 +1,426 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
+#include <random>
+
+#include "ApparentHorizons/SpherepackIterator.hpp"
+#include "ApparentHorizons/Strahlkorper.hpp"
+#include "Numerical/RootFinding/QuadraticEquation.hpp"
+#include "tests/Unit/ApparentHorizons/YlmTestFunctions.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+namespace {
+
+void test_invert_spec_phys_transform() {
+  const double avg_radius = 1.0;
+  const double delta_radius = 0.1;
+  const size_t l_grid_init = 33;
+  const auto l_grid = static_cast<size_t>(l_grid_init * 1.5);
+  const std::array<double, 3> center = {{0.1, 0.2, 0.3}};
+
+  // Create radius as a function of angle
+  DataVector radius(YlmSpherepack::physical_size(l_grid_init, l_grid_init),
+                    avg_radius);
+  {
+    std::uniform_real_distribution<double> ran(0.0, 1.0);
+    std::mt19937 gen;
+    for (auto& r : radius) {
+      r += delta_radius * ran(gen);
+    }
```

Comment:
Dunno. Anyone else?

Reply:
I don't think it's necessary here since the RNG is unseeded and so the test will use the same numbers each time. We use CAPTURE_PRECISE when the RNG is seeded so catch will output the values that it failed with.

Reply:
We do consider this good practice.  Note that CAPTURE_PRECISE is scoped and will only display if it is still in scope when a CHECK fails, so don't do it inside the braces scoping the RNG.  I believe CAPTURE_PRECISE will work on an entire DataVector, so that should be easy here.

Reply:
I stand corrected.  A default-constructed RNG is deterministic as @osheamonn says, and so does not need to be captured.